src/chapter2.adoc

```diff
@@ -104,23 +104,33 @@ The table format is described in <<RQSC_TABLE>>.
 |===
 |Field {set:cellbgcolor:green}|Byte Length|Byte Offset|Description
 4+<|{set:cellbgcolor:!} *Header*
-<|- Resource Type               |1  |0 <a|
+<|- Resource Type               |1  |0  <a|
                                             - 0 - Cache
                                             - 1 - Memory
                                             - 2 - 0x7F - Reserved
                                             - 0x80 - 0xFF - Vendor Specific
-<|- Length                      |1  |1 <a| Length of this specific Resource Structure.
+<|- Reserved                    |1  |1  <a|Reserved. Must be 0
+<|- Length                      |2  |2  <a| Length of this specific Resource Structure.
                                             Length includes the Resource Specific
-                                            Data bytes as well. If length is set to 16,
+                                            Data bytes as well. If length is set to 20,
                                             then, it indicates there is no resource
                                             specific data available for this structure.
-<|- Resource Flags              |2  |2 <a|Resource Type Specific flags. Refer to
+<|- Resource Flags              |2  |4  <a|Resource Type Specific flags. Refer to
                                             <<RESOURCE_FLAGS_TABLE>> for details.
-<|- Resource ID 1               |8  |4 <a|Depends on the Resource Type field. Refer
+<|- Reserved                    |1  |6  <a|Reserved. Must be 0
+<|- Resource ID Type            |1  |7  <a|Type of the Resource ID described by
+                                            Resource ID 1 and Resource ID 2 fields.
+
+                                            - 0 - Cache ID
```

Comment:
Thanks.. Since the field name itself is "Resource ID Type", Just "Cache" is sufficient IMO. The description of Resource ID tells it is Cache ID to look in the PPTT.

Regarding "Resource Type" vs "Resource ID Type",  what if some one sets "Resource Type" as Cache and "Resource ID Type" as ACPI instead of Cache? Is that a valid configuration?  If valid, how does it find corresponding entry in PPTT? If not valid, then why not define it in Resource Type itself?

IIRC, the feedback from last meeting was to elaborate the  reason for Resource ID 1 to be 8 bytes with ACPI HID use case. But IMHO,  it is not required to explain the reason it in the spec. It can be simply for future use as well. Current changes in this PR look more confusing to me.

Reply:
Yes, it would be a valid configuration as long as the Cache associated with a Resource ID Type of ACPI is indeed represented via an ACPI Device and not a PPTT structure.

I heard the feedback as to actually accommodate the exact statement above, which is being able to identify the ID of a resource in different ways than a set way, mainly because the implementation of a QoS controller may target many different types of devices.

I'll clean up the feedback you have given so far in terms of removing unnecessary words and let's see if it makes it better to understand.